tasks/100_traverse-tree/traverse-tree.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * traverseTree - Perform recursive traverse of tree.
+ *
+ * @param {Object} node - current node of tree.
+ * @param {Function} adder - executes on each node.
+ */
+export function traverseTree(node, adder) {
+  for (let i = 0; i < node.children.length; i++) {
+    traverseTree(node.children[i], adder);
+  }
+  adder(node);
```

Comment:
Why don't you call `adder` on node before you start loop?

Reply:
@KalachevDev we don't care about traverse order because in tests result array will be sorted